proposals/partial-properties.md

```diff
@@ -0,0 +1,142 @@
+# Partial properties
+https://github.com/dotnet/csharplang/issues/6420
+
+### Grammar
+
+The *property_declaration* grammar [(§14.7.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1471-general) is updated as follows:
+
+```diff
+property_declaration
```

Comment:
Have we discussed whether the change should extend to indexers too (that feels like it should go along with properties), and possibly events as well?